test/readme-test/run-readme-spellcheck

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
-set -euo errexit
+set -euo pipefail
```

Comment:
did `shellcheck` recommend these??

Reply:
it recommended some of the other changes I made in the scripts. But this one was me poking around and noticing.